docs/samples/contact-center/app.js

```diff
@@ -103,14 +106,50 @@ function initWebex(e) {
 
 credentialsFormElm.addEventListener('submit', initWebex);
 
+
 function register() {
     webex.cc.register().then((data) => {
         console.log('Event subscription successful: ', data);
-    }).catch(() => {
-        console.log('Event subscription failed');
+        const agentProfile = webex.cc.getAgentProfileData();
+        teamsDropdown.innerHTML = ''; // Clear previously selected option on teamsDropdown
+        const listTeams = agentProfile.teamsList;
+        listTeams.forEach((team) => {
+          if(team.teamType === "AGENT") {
+            const option = document.createElement('option');
+            option.value = team.id;
+            option.text = team.name;
+            teamsDropdown.add(option);
+          }
+        });
+        const loginVoiceOptions = agentProfile.agentDesktopProfile.loginVoiceOptions;
+        agentLogin.innerHTML = '' // Clear previously selected option on agentLogin.
+        if(loginVoiceOptions.length > 0) agentLoginButton.disabled = false;
+        loginVoiceOptions.forEach((voiceOptions)=> {
+          const option = document.createElement('option');
+          option.text = voiceOptions;
+          option.value = voiceOptions;
+          agentLogin.add(option);
+        });
+    }).catch((error) => {
+        console.log('Event subscription failed', error);
     })
 }
 
+async function handleAgentLogin(e) {
+  const value = e.target.value;
+  if (value === 'Desktop') {
+    agentDeviceType = 'BROWSER';
+    deviceId = 'webrtc-6b310dff-569e-4ac7-b064-70f834ea56d8'
+  } else {
+    agentDeviceType = 'EXTENSION';
+    deviceId = '1001'
+  }
+}
+
+function doAgentLogin() {
+  webex.cc.doAgentLogin(teamsDropdown.value, agentDeviceType, deviceId);
```

Comment:
Where is this function? I couldn't find this in cc.ts file Also, API invocation should look something like webex.cc.agent.stationLogin()